### PR TITLE
nrf_ppi: Add functions to modify only the event or task endpoints

### PR DESCRIPTION
**Pull request by aescolar**
Added 2 new functions: one for modifying only the event endpoint,
the other to modify only the task endpoint for a given channel.

Signed-off-by: Alberto Escolar Piedras <alpi@oticon.com>

### CODE REVIEW

File: hal/nrf_ppi.h [hunk 1]
```diff
@@ -392,6 +413,18 @@ __STATIC_INLINE void nrf_ppi_channel_endpoint_setup(nrf_ppi_channel_t channel,
     NRF_PPI->CH[(uint32_t) channel].TEP = tep;
 }
 
+__STATIC_INLINE void nrf_ppi_channel_set_event_endpoint(nrf_ppi_channel_t channel,
+                                                        uint32_t          eep)
+{
+    NRF_PPI->CH[(uint32_t) channel].EEP = eep;
```
[the remark this answers is not in the document]
Comment (aescolar) [reply hal/nrf_ppi.h]: Thanks @cvinayak , that line is just copy pasted from `nrf_ppi_channel_endpoint_setup()`

File: hal/nrf_ppi.h [hunk 2]
```diff
@@ -214,6 +214,27 @@ __STATIC_INLINE void nrf_ppi_channel_endpoint_setup(nrf_ppi_channel_t channel,
                                                     uint32_t          eep,
                                                     uint32_t          tep);
 
+/**
+ * @brief Function for setting up event endpoint for a given PPI channel.
+ *
+ * @param[in] eep Event register address.
+ *
```
Comment (kl-cruz) [inline hal/nrf_ppi.h]: This new line can be removed

File: hal/nrf_ppi.h [hunk 3]
```diff
@@ -214,6 +214,27 @@ __STATIC_INLINE void nrf_ppi_channel_endpoint_setup(nrf_ppi_channel_t channel,
...
+ * @param[in] channel Channel to which the given endpoints are assigned.
+ */
+__STATIC_INLINE void nrf_ppi_channel_set_event_endpoint(nrf_ppi_channel_t channel,
```
Comment (kl-cruz) [inline hal/nrf_ppi.h]: verb should be placed at the end of the function's name: nrf_ppi_channel_event_endpoint_set

Comment (kl-cruz) [reply hal/nrf_ppi.h]: same comment to set_task function

File: hal/nrf_ppi.h (hunk 2 above)
Comment (kl-cruz) [reply hal/nrf_ppi.h]: same comment to set_task function

**Comment by aescolar** [comment]
@kl-cruz : Thanks, done

**Comment by aescolar** [comment]
@anangl : Done also.

**Pull request merged**